terraform/layer1-aws/aws-eks.tf

```diff
@@ -268,7 +207,7 @@ module "vpc_cni_irsa" {
 
 module "aws_ebs_csi_driver" {
   source  = "terraform-aws-modules/iam/aws//modules/iam-role-for-service-accounts-eks"
-  version = "4.14.0"
+  version = "4.17.0"
```

Comment:
Why do you use different versions for the same module? See the `vpc_cni_irsa` terraform module

Reply:
i'm sorry it's my mistake, fixed